### PR TITLE
docs(use-cases): add use cases entry in the doc website and start adding content

### LINKED ISSUE #1568: Port over qf super site content to maci.pse.dev

### LINKED ISSUE #1569: Setup maci use cases entry in the documentation website

### PR DESCRIPTION
**Pull request by ctrlc03**
# Description

Port over qf super site content and bootstrap maci use cases entry.

## Related issue(s)

fix #1569 
fix #1568 

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: website/versioned_docs/version-v2.0_alpha/use-cases/public-goods-funding/quadratic-funding/what-is-quadratic-funding.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+Quadratic funding is made up of:
+
+1. a pool of funds to be distributed
+2. a voting system for distributing those funds
+
+Two common voting systems you may have used are 1-person-1-vote and 1-dollar-1-vote. Quadratic funding is a best-of-both-worlds solution where users vote with the **square root** of the money they spend — read on to learn more.
+
+# **Problems with Existing Funding Mechanisms**
+
+The common fund-distribution systems of today all have pros and cons:
```
Comment (kittybest) [inline website/versioned_docs/version-v2.0_alpha/use-cases/public-goods-funding/quadratic-funding/what-is-quadratic-funding.md:25]: maybe: ***All common fund-distribution systems have pros and cons*** ?

File: website/versioned_docs/version-v2.0_alpha/use-cases/public-goods-funding/quadratic-funding/how-it-works.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+Requirements here vary widely depending on the structure of the round & team size.
+
+This includes technical & operational work:
+
+- **Technical work**
+  Eg. Configuring wallets, deploying contracts & deploying the website
+  We’d recommend allocating approximately 200 hours of developer time to prepare for a round.
+- **Operational work**
+  Marketing, communications, curating & reviewing project applications & community support
```
Comment (kittybest) [inline website/versioned_docs/version-v2.0_alpha/use-cases/public-goods-funding/quadratic-funding/how-it-works.md:43]: image

⬆️ looks like this, maybe adding a period or a colon after topic?

**Review by kittybest — COMMENTED** [review]
image

Not sure why both `Governance` and `Maci in Governance` exists, and why double layer of `Local Elections` --> will there be other use cases under `Maci in Governance` and `Local Elections`?

**Comment by ctrlc03** [comment]
> Not sure why both Governance and Maci in Governance exists, and why double layer of Local Elections --> will there be other use cases under Maci in Governance and Local Elections?

so both governance and local elections are placeholders for now, we don't have any content to add there but hopefully soon. Can maybe rename the individual md entry to introduction 🤔

File: website/versioned_docs/version-v2.0_alpha/use-cases/public-goods-funding/retroactive-public-goods-funding/how-it-works.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+Requirements here vary widely depending on the structure of the round & team size.
+
+This includes technical & operational work:
+
+- **Technical work**
+  Eg. Configuring wallets, deploying contracts & deploying the website
+  We’d recommend allocating approximately 200 hours of developer time to prepare for a round.
+- **Operational work**
+  Marketing, communications, curating & reviewing project applications & community support
```
Comment (kittybest) [inline website/versioned_docs/version-v2.0_alpha/use-cases/public-goods-funding/retroactive-public-goods-funding/how-it-works.md:34]: image

also add a `-` after the title?

File: website/versioned_docs/version-v2.0_alpha/use-cases/public-goods-funding/retroactive-public-goods-funding/how-it-works.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+## 1. Apply
+
+### Application
+
+- First step is to submit a proposal outlining your RPGF plans & how you plan to use the funds from the grant application: [Apply here](https://quadraticfunding.paperform.co/)
```
Comment (kittybest) [inline website/versioned_docs/version-v2.0_alpha/use-cases/public-goods-funding/retroactive-public-goods-funding/how-it-works.md:40]: should the link here same as the qf one?

**Review by kittybest — COMMENTED** [review]
thanks! just few comments

File: website/versioned_docs/version-v2.0_alpha/use-cases/public-goods-funding/retroactive-public-goods-funding/registration-approaches.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+---
+title: Retroactive public goods funding - Registration approaches
```
Comment (kittybest) [inline website/versioned_docs/version-v2.0_alpha/use-cases/public-goods-funding/retroactive-public-goods-funding/registration-approaches.md:2]: adding title here should automatically add a title on the top of the page, but here the title is not shown:
image

File: website/versioned_docs/version-v2.0_alpha/use-cases/public-goods-funding/retroactive-public-goods-funding/how-it-works.md (hunk 4 above)
Comment (ctrlc03) [reply website/versioned_docs/version-v2.0_alpha/use-cases/public-goods-funding/retroactive-public-goods-funding/how-it-works.md:38]: for now

**Review by kittybest — APPROVED** [review]
Thanks!

**Pull request merged**